sdk/forms/src/main/java/com/klaviyo/forms/InAppForms.kt

```diff
@@ -41,3 +44,28 @@ fun Klaviyo.registerForInAppForms(
     // And initialize the webview client
     Registry.get<WebViewClient>().initializeWebView()
 }
+
+/**
+ * Halts the in-app forms services and observers,
+ * hiding any currently displayed forms and preventing any further forms from being presented.
+ */
+@UiThread
+fun Klaviyo.unregisterInAppForms() = safeApply {
+    Registry.apply {
+        unregister<InAppFormsConfig>()
+        getOrNull<PresentationManager>()?.dismiss()
+        getOrNull<WebViewClient>()?.destroyWebView()
+    }
+}
```

Comment:
Unregister refers to the in-app observers and triggering logic, doesn't mean we need to unregister the underlying services. Just stop them using up resources -- finish activity if running, destroy the webview, stop the observers, and it also makes sense to discard the registered config.